docs/modules/debug/require-inspector.md

```diff
@@ -0,0 +1,28 @@
+<%( TOC )%>
+
+# Require inspector extension
```

Comment:
I'm not sure about this name. Need to think about it a bit

Reply:
removed the extension bit here and in the TOC